### PR TITLE
App Submission: Homebridge v2022.6.29

### PR DESCRIPTION
**Pull request by johnpc**
# App Submission

### App name

Homebridge

### 256x256 SVG icon

https://jpc.io/r/homebridge.svg

icon_adobe_express

...

### Gallery images

Screen Shot 2022-06-29 at 1 02 47 AM
Screen Shot 2022-06-29 at 1 03 03 AM
Screen Shot 2022-06-29 at 1 03 22 AM

...

### I have tested my app on:
- [ ] [Umbrel dev environment](https://github.com/getumbrel/umbrel-dev)
- [x] [Umbrel OS on a Raspberry Pi 4](https://github.com/getumbrel/umbrel-os)
- [ ] [Custom Umbrel install on Linux](https://github.com/getumbrel/umbrel#-installation)

### CODE REVIEW

**Comment by johnpc** [comment]
I've been using Homebridge to connect my Nest thermostat to Apple HomeKit. It's pretty simple to set up outside of Umbrel, but it would be nice to have to in the App Store with the rest of the tools I run on the device.

File: homebridge/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_HOMEBRIDGE_WEB_IP
+      APP_PORT: $APP_HOMEBRIDGE_WEB_PORT
+  homebridge:
+    image: oznu/homebridge:ubuntu
+    container_name: homebridge
+    restart: always
```
Comment (nevets963) [inline homebridge/docker-compose.yml:10]: We currently rely on the `on-failure` restart policy. ie only restarting if the container unexpectedly stops as opposed to always restarting.

**Review by nevets963 — COMMENTED** [review]
@johnpc Thank you for the app submission!!

I've left some feedback/questions on the PR :)

File: homebridge/docker-compose.yml (hunk 1 above)
Comment (johnpc) [reply homebridge/docker-compose.yml]: sounds good!

File: homebridge/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    network_mode: host
+    environment:
+      - HOMEBRIDGE_CONFIG_UI_PORT=8581
+    volumes:
+      - ./homebridge:/homebridge
```
[the remark this answers is not in the document]
Comment (johnpc) [reply homebridge/docker-compose.yml]: makes sense

File: homebridge/umbrel-app.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: homebridge
+category: Automation
+name: Homebridge
+version: "2022.6.29"
```
[the remark this answers is not in the document]
Comment (johnpc) [reply homebridge/umbrel-app.yml]: :+1:

**Comment by lukechilds** [comment]
Hey @johnpc just a ping on this: https://github.com/getumbrel/umbrel-apps/pull/53/files#r913493846

Is it possible to remove `network_mode: host`? It's invalid to have an internal assigned IP in the `networks` block and also have the container using host networking. It also means we can't front the app with Umbrel's auth proxy if it's running directly on the host.

Does Homebridge work without access to host networking or is it a hard requirement?

**Comment by johnpc** [comment]
It does work, but I am abandoning this PR. I don't think the Homebridge software is friendly enough for Umbel use case. I've ran into issues with it lately. I worry it will only clutter Umbrel's App Store.

**Pull request closed without merging**